tests/unit/src/test/scala/tests/codeactions/ConvertToNamedArgumentsLspSuite.scala

```diff
@@ -24,6 +24,7 @@ class ConvertToNamedArgumentsLspSuite
        |  case class Foo(param1: Int, param2: Int, param3: Int)
        |  Foo(param1 = 1, param2 = 2, param3 = 3)
        |}""".stripMargin,
+    filterAction = filterAction,
```

Comment:
Could you add the possible actions instead? I think it would be good to see all the possible actions to make sure that they don't pop up in unexpected places.